adyenofficial.php

```diff
@@ -522,7 +522,7 @@ public function displayGetStarted()
      */
     public function displayForm()
     {
-        $this->context->controller->addCSS($this->_path . 'views/css/adyen_admin.css', 'all');
+        $this->context->controller->addCSS('modules/' . $this->name . '/views/css/adyen_admin.css', 'all');
```

Comment:
shall we make this a constant and reuse that one on all the places ?

Reply:
I would keep it this way because I need to use $this->name in the value so I could create a static property maybe.